web/Controllers/LandlordController.cs

```diff
@@ -51,29 +57,47 @@ public async Task<ActionResult> RegisterPost([FromForm] LandlordProfileModel cre
         }
 
         var user = GetCurrentUser();
+        ILandlordService.LandlordRegistrationResult result;
+        LandlordDbModel? landlord = null;
 
-        var result = await _landlordService.RegisterLandlordWithUser(createModel, user);
+        if (createModel.Unassigned && user.IsAdmin)
+        {
+            (result, landlord) = await _landlordService.RegisterLandlord(createModel);
+        }
+        else
+        {
+            return StatusCode(403);
+        }

```

Comment:
Still need to handle the regular registration case!

Reply:
Oh dear, there isn't a test for it is there? That's why they all still passed